webapp/channels/src/components/admin_console/setting.tsx

```diff
@@ -5,15 +5,15 @@ import React, {PureComponent} from 'react';
 
 import SetByEnv from './set_by_env';
 
-type Props = {
+export type SettingsProps = {
```

Comment:
i am sorry for not being clear last time, but this can be Props, but only while we import it in remove_file_setting we can change the Props name. https://www.typescriptlang.org/docs/handbook/2/modules.html#additional-import-syntax

```ts
import type {Props as SettingsProps} ....
```